dashboard2/cookbook/whitelabel.md

```diff
@@ -76,13 +76,10 @@ the appropriate data.
     
 7. In order for the translation proxy to be able to send emails from under your
    domain, you will need to provide authorization to the email
-   service. This is done by adding specialized DNS records.
-
- 	1. `mandrill._domainkey.${APP_DOMAIN}` TXT `v=DKIM1; k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCrLHiExVd55zd/IQ/J/mRwSRMAocV/hMB3jXwaHH36d9NaVynQFYV8NaWi69c1veUtRzGt7yAioXqLj7Z4TeEUoOLgrKsn8YnckGs9i3B3tVFB+Ch/4mPhXWiNfNdynHWBcPcbJ8kjEQ2U8y78dHZj1YeRXXVvWob2OaKynO8/lQIDAQAB;`
-    
-	2. `${SELECTOR}._domainkey.${APP_DOMAIN}` TXT `${DKIM_KEY}` - these values will be provided to you as they are domain-dependent.
+   service. This is done by adding a specialized DNS record:
     
-	3. `v=spf1 include:spf.mandrillapp.com include:sparkpostmail.com ?all`; or if you're already using an SPF record, add `include:spf.mandrillapp.com include:sparkpostmail.com` just before the last operator.
```

Comment:
Ideally, this should remain - SPF is important, we should only remove Mandrill, but retain the instruction to add Spark Post.